Launch_multiple_spm.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
Is it good practice to have a filename starting with uppercase in Python?

Reply:
True, need to lower the case for this file.